src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,22 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private static final int INITIAL_SIZE = 5;
+    private final List<Bulldozer> bulldozers;
+
+    public BulldozerProducer() {
+        bulldozers = new ArrayList<>();
+    }
+
+    @Override
+    public List<Bulldozer> get() {
```

Comment:
Okay, but if you call get multiple times on the same object. Will add 5 elements each time, instead of generating new 5.

Reply:
Done